fpdf/line_break.py

```diff
@@ -0,0 +1,237 @@
+from collections import namedtuple
+
+SOFT_HYPHEN = "\u00ad"
+HYPHEN = "\u002d"
+SPACE = " "
+NEWLINE = "\n"
+
+
+class Fragment:
+    def __init__(self, style, underlined, characters=None):
+        self.characters = [] if characters is None else characters
+        self.style = style
+        self.underline = underlined
+
+    @classmethod
+    def from_string(cls, string, style, underlined):
+        return cls(style, underlined, list(string))
+
+    def add_character(self, character):
```

Comment:
This method is not used and can be removed

Reply:
Will do